lambdas/checklist/client/github.py

```diff
@@ -176,7 +220,8 @@ def get_important_file_names_by_languages(languages: dict) -> list:
         :param languages: A list of programming languages (and their usage details) used in the repository.
         :return: A list of file names that are important for the given programming languages.
         """
-        important_file_names = ["README.md"]
+        important_file_names = []
+        # important_file_names.append("README.md")
```

Comment:
We're not adding README.md anymore?

Reply:
to reduce the size of the prompt (README file is hard to compress and it gives unnecessary info)

Reply:
Do we want to remove it then or we readd this later at a certain point?